tests/commands/package/test_call.py

```diff
@@ -232,14 +190,14 @@ def test_identity_arg_package_one_app(package_command, first_app, tmp_path):
 
 
 def test_adhoc_sign_package_one_app(package_command, first_app, tmp_path):
-    """If there is one app,and an --adhoc argument, package signs the app using adhoc
+    """If there is one app,and an --adhoc argument, package signs the app using ad-hoc
     option."""
     # Add a single app
     package_command.apps = {
         "first": first_app,
     }
 
-    # Configure an adhoc signing option
+    # Configure an ad-hoc signing option
     options = package_command.parse_options(["--adhoc"])
```

Comment:
I'm not sure how this passed previously... the argument should be `--adhoc-sign`